portality/formcontext/xwalk.py

```diff
@@ -917,7 +917,8 @@ def obj2form(cls, obj):
 
         forminfo['subject'] = []
         for s in bibjson.subjects():
-            forminfo['subject'].append(s['code'])
+            if "code" in s:
```

Comment:
when I was demonstrating the CSV generation to @richard-jones he got an error on his local index and added this line to fix.